scripts/globals/items/plate_of_tentacle_sushi_+1.lua

```diff
@@ -39,15 +39,14 @@ end;
 -----------------------------------
 
 function onEffectGain(target,effect)
-	target:addMod(MOD_HP, 20);
-	target:addMod(MOD_DEX, 3);
-	target:addMod(MOD_AGI, 3);
-	target:addMod(MOD_MND, -1);

```

Comment:
MND is legit according to ffxiclopedia
http://ffxiclopedia.wikia.com/wiki/Tentacle_Sushi_%2B1 (applies to regular tentacle sushi too)